addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -4139,7 +4139,7 @@ var BasicModel = AbstractModel.extend({
                     }
                 } else {
                     dataPoint = self._makeDataPoint({
-                        context: list.context,
+                        context: list.getContext(),
```

Comment:
It's only a master fix ? Not for 12.0 at least ?

Reply:
i wouldn't risk this in 12 ^^

Reply:
There is no risk, we got your tests ! :p Anyway, your call